ControlRoomApplication/ControlRoomApplicationTest/EntitiesTests/AppointmentCalibrationTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
I think this whole file is unnecessary and can be removed.

- [ ] Remove unnecessary test